src/Sylius/Bundle/AttributeBundle/Form/Type/AttributeType/Configuration/OptionListAttributeConfigurationType.php

```diff
@@ -32,13 +32,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('format', 'choice', [
                 'choices' => [
-                    self::FORMAT_DROPDOWN           => 'Dropdown',
-                    self::FORMAT_MULTIPLE_SELECTION => 'Multiple selection',
+                    self::DROPDOWN    => 'Dropdown',
+                    self::MULTISELECT => 'Multiple selection',
                 ],
                 'attr' => [
                     'class' => 'js-format-selector'
                 ],
             ])
+            ->add('expanded', 'checkbox', [
+                'attr' => [
+                    'description' => 'This will dictate if using radio buttons (Dropdown) or multi checkboxes (Multiple selection).'
```

Comment:
you're in the Sylius repo, I only added this to Reiss.  

But even then, I'm not sure if it works on checkboxes